solutions/silver/cf-104536F.mdx

```diff
@@ -35,13 +35,12 @@ using namespace std;
 int max_length = 0, max_node;
 vector<vector<int>> tree1, tree2;
 
+// Computes the farthest node from the node passed to the root call, and the length of the path to it
 void dfs(const vector<vector<int>> &tree, int node, int previous = 0,
```

Comment:
i feel like `max_length` and `max_node` can be modified to be return values instead of global variables, it makes the flow of data in the program clearer

Reply:
Good suggestion, looks much nicer now.